src/store_swapin.cc

```diff
@@ -56,7 +56,7 @@ storeSwapInFileClosed(void *data, int errflag, StoreIOState::Pointer)
 
     if (sc->_callback.pending()) {
         assert (errflag <= 0);
-        sc->callback(0, errflag ? true : false);
+        sc->noteSwapInDone(errflag ? true : false);
```

Comment:
```suggestion
        sc->noteSwapInDone(errflag);
```

Reply:
I agree, but I think we should minimize the code change here until others request that we do this out-of-scope polishing. If we start polishing this proactively, we are likely to trigger more (out of scope) change requests IMO.